scripts/validate_agent_prompts.py

```
@@ -0,0 +1,264 @@
+
+    errors: list[str] = []  # Blocking errors (fail CI)
+    warnings: list[str] = []  # Non-blocking warnings
+    all_agents = get_all_agent_names()
+    all_skills = get_all_skill_names()
+
+    for agent_path in sorted(AGENTS_DIR.glob("*.md")):
+        rel_path = agent_path.relative_to(ROOT)
+
+        try:
+            _, content = extract_frontmatter_and_content(agent_path)
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)